### PR TITLE
refactor-for-DataGridCellInput

### PR DESCRIPTION
**Pull request by barteksiudak**
refactor-for-DataGridCellInput

### CODE REVIEW

File: src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js [hunk 1]
```diff
@@ -10,147 +17,91 @@ import './DatagridCellInput.scss';
 const ENTER_KEY_CODE = 13;
 const ESC_KEY_CODE = 27;
 
-class DatagridCellInput extends React.Component {
-  constructor(props) {
-    super(props);
+const DatagridCellInput = props => {
```
Comment (kevinmlsilva) [inline src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js]:
```suggestion
const DatagridCellInput = ({ editing,
    modifiers,
    onChange,
    onStopEditing,
    onStartEditing,
    value,
    type}) => {
```

File: src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js [hunk 2]
```diff
@@ -10,147 +17,91 @@ import './DatagridCellInput.scss';
...
-      onKeyDown: this.handleKeyDown,
-      onChange: this.handleChange,
+  function getInputProps() {
+    const inputProps = {
+      ...props,
+      ref: inputRef,
+      handleChange: onChange,
+      handleChangeFromOverlay,
+      handleStartEditing,
+      handleBlur,
```
Comment (NicolasMartos) [inline src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js]:
```suggestion
      handleBlur: onStopEditing,
```

**Comment by NicolasMartos** [comment]
I'm not sure about `DatagridCellInputAdapaters` name. Could we put all this getProps function in `helpers`?

File: src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js (hunk 1 above)
Comment (barteksiudak) [reply src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js]: Sorry, but then tests are failed.
In some cases we need eg. `onChange` and `handleChange`. 
After destructurization we loose them.
We need each prop for `getInputProps`

**Comment by barteksiudak** [comment]
> I'm not sure about `DatagridCellInputAdapaters` name. Could we put all this getProps function in `helpers`?

@NicolasMartos IMO we need some more abstraction.
Push everything to `helpers` is a bit strange.
For now helpers are also: utils, services, factories, modules and adapters in one - like a trash :(

So, should I change proposed `adapters` sufix or leave it ?

**Pull request merged**